main.css

```diff
@@ -33,36 +33,121 @@ body {
   padding-bottom: 5px;
 }
 .post-body {
-  max-width: 650px;
+  max-width: 650x;
 }
 
 .sidebar-container {
   border: 3px solid lightseagreen;
   margin: 20px;
 }
+.sidebar-content {
+  background-color: red;
+}
 
 .sidebar-quote {
   text-align: center;
 }
-.sidebar-content {
-  padding: 5px;
+.sidebar-nav {
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.hidden {
+  display: none;
 }
 
 @media (min-width: 600px) {
   .post-container {
     margin: 20px 10px 20px 10px;
   }
-
-  .all-posts-page {
+  .home-page {
     display: flex;
   }
   .sidebar-container {
     width: 200px;
+    height: 400px;
+  }
+
+  .all-posts-page {
+    display: flex;
+  }
+
+  .post-body {
+    max-width: 650px;
   }
   .post-title {
     text-align: center;
   }
   .all-posts {
     max-width: 600px;
   }
+  .hidden {
+    display: none;
+  }
+}
+
+/* contact page form....................................................................... */
+h1 {
```

Comment:
these element selectors (h1, h3, form, and label) should go before all the class selectors. they're more generic than class or ID selectors, so i would expect any basic styles for a `<form>` element, to be possibly overridden by a class selector for a particular form, but that won't happen if these declarations come after the classes